devdoc/CodeSubmissions.md

```diff
@@ -0,0 +1,71 @@
+# Code Submission Guidelines
+
+We welcome submissions to the platform especially in the form of pull requests into the HPCC-Systems github repository.  The following describes some of process for merging PRs.
+
+## Pull requests
+There are a few things that should be considered when creating a PR to increase the likelihood that they can be accepted quickly.
+
+* Write a good commit message\
```

Comment:
commit messages should reference a specific JIRA whenever possible,  Even if it's an update due to code review, it should reference the issue number or at the very least be more descriptive.

Reply:
I'm not sure that follow on commits need to the issue number - unless they are going to be preserved when the change is merged.  If they will be squashed then it is ok to say "changes following review"